WunderSys.xml

```diff
@@ -34803,12 +34803,14 @@ end
 							<string>^[A-Z][a-z]+ slashes into your (.+) with</string>
 							<string>^[A-Z][a-z]+ swings .+ at your (.+) with all (?:his|her) might\.$</string>
 							<string>^Lightning\-quick, [A-Z][a-z]+ (?:jabs|stabs) your (.+) with .+\.$</string>
+							<string>^[A-Z][a-z]+ unleashes a terrible blow at your (.+) with a .+\.$</string>
```

Comment:
I think this is already covered by the Hew trigger that also tracks fractures (it calls wsys.my_limb_hit_add in the prompt check). I think we probably need to remove that call then? I agree it makes more sense to group the limb hit tracking together, rather than having it part of the aff tracking (which I'm removing anyway). It might make sense to still call it out as a 2h attack, rather than generic "weapon" attack.

Reply:
If you want to just verify this doesn't cause double counting with 2h hew, or else revert just this piece, I can go ahead and move all the 2h limb hit tracking here while I'm making the aff tracking changes.